DESCRIPTION

```diff
@@ -8,8 +8,8 @@ Authors@R: c(
         person("Florian", "Hahne", role = "aut"),
         person("Emmanuel", "Taiwo", role = "ctb",
                 comment = "'howtogenefinder' vignette translation from Sweave to RMarkdown / HTML."),
-        person("Bioconductor Package Maintainer", role = "cre",
-                email = "maintainer@bioconductor.org"))
+        person("Khadijah", "Amusat", role = "ctb",
+                comment = "Converted genefilter vignette from Sweave to RMarkdown / HTML."),
```

Comment:
Be careful here. We should keep the Bioconductor Package Maintainer. You also should notice that, looking at line 4, that this is a vector, so you need a closing parenthesis and items should be separated by commas.
Add back in the Bioconductor package maintainer, separate the person entries with a comma, and replace the comma on line 12 with a parenthesis.